plugins/woocommerce/src/Internal/Admin/Events.php

```diff
@@ -123,6 +127,23 @@ final public static function instance() {
 	public function init() {
 		add_action( 'wc_admin_daily', array( $this, 'do_wc_admin_daily' ) );
 		add_filter( 'woocommerce_get_note_from_db', array( $this, 'get_note_from_db' ), 10, 1 );
+		add_action(
+			'woocommerce_plugins_install_before',
+			function ( $slug, $source ) {
+				$this->install_options_for_core_profiler_plugin_install( $slug, $source, 'before' );
+			},
+			10,
+			2
+		);
+
+		add_action(
+			'woocommerce_plugins_install_after',
+			function ( $slug, $source ) {
+				$this->install_options_for_core_profiler_plugin_install( $slug, $source, 'after' );
+			},
+			10,
+			2
+		);
```

Comment:
I was curious if there is a specific reason why these actions are being added here instead of in OnboardingSetup Wizard.php?

Reply:
@chihsuan Good catch! I completely forgot about OnboardingSetupWizard.php class 🤦 I'll move it to there.

Reply:
@chihsuan Fixed it in [Move woocommerce_plugins_install_before and woocommerce_plugins_insta…](https://github.com/woocommerce/woocommerce/pull/56373/commits/be71be75ce9b082361cc185cc82b48b3ae5017e8)